concourse/steps/scan_container_images.py

```diff
@@ -476,3 +433,59 @@ def print_protecode_info_table(
         ('Component name filter (exclude)', exclude_component_names),
     )
     print(tabulate.tabulate(entries, headers=headers))
+
+
+def virus_scan_images(
```

Comment:
hmm... not sure whether this really belongs into this module. let us discuss this later (in our next sync)